Add new docker image for rultor with docker

For running integration test we need docker in running docker image in rultor.

```
[ERROR] Failed to execute goal io.fabric8:docker-maven-plugin:0.26.1:start (start) on project seeds-db: Execution start of goal io.fabric8:docker-maven-plugin:0.26.1:start failed: No <dockerHost> given, no DOCKER_HOST environment variable, no read/writable '/var/run/docker.sock' or '//./pipe/docker_engine' and no external provider like Docker machine configured -&gt; [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal io.fabric8:docker-maven-plugin:0.26.1:start (start) on project seeds-db: Execution start of goal io.fabric8:docker-maven-plugin:0.26.1:start failed: No <dockerHost> given, no DOCKER_HOST environment variable, no read/writable '/var/run/docker.sock' or '//./pipe/docker_engine' and no external provider like Docker machine configured
```

--- a/.rultor.yml
+++ b/.rultor.yml
@@ -1,5 +1,6 @@
 docker:
   image: yegor256/java8
+  as_root: true
 decrypt:
   settings.xml: "repo/.rultor/settings.xml.asc"
   pubring.gpg: "repo/.rultor/pubring.gpg.asc"

--- a/seeds-db/pom.xml
+++ b/seeds-db/pom.xml
@@ -77,9 +77,8 @@ SOFTWARE.
                   <name>healthcheck/postgres:alpine</name>
                   <alias>postgres</alias>
                   <run>
-                    <ports>
-                      <port>postgres.port:5432</port>
-                    </ports>
+                    <exposedPropertyKey>postgres</exposedPropertyKey>
+                    <namingStrategy>alias</namingStrategy>
                     <env>
                       <POSTGRES_PASSWORD>mysecret</POSTGRES_PASSWORD>
                       <POSTGRES_USER>testuser</POSTGRES_USER>
@@ -100,6 +99,20 @@ SOFTWARE.
                   <goal>start</goal>
                 </goals>
               </execution>
+              <execution>
+                <id>stop</id>
+                <phase>post-integration-test</phase>
+                <goals>
+                  <goal>stop</goal>
+                </goals>
+              </execution>
+              <execution>
+                <id>remove</id>
+                <phase>post-integration-test</phase>
+                <goals>
+                  <goal>remove</goal>
+                </goals>
+              </execution>
             </executions>
           </plugin>
           <plugin>
@@ -107,7 +120,7 @@ SOFTWARE.
             <artifactId>maven-failsafe-plugin</artifactId>
             <configuration>
               <systemPropertyVariables>
-                <jdbs.url.postgres>jdbc:postgresql://localhost:${postgres.port}/testuser</jdbs.url.postgres>
+                <jdbs.url.postgres>jdbc:postgresql://${docker.container.postgres.ip}:5432/testuser</jdbs.url.postgres>
               </systemPropertyVariables>
             </configuration>
           </plugin>

--- a/test-steps/pom.xml
+++ b/test-steps/pom.xml
@@ -41,7 +41,7 @@ SOFTWARE.
       <groupId>com.github.mkolisnyk</groupId>
       <artifactId>cucumber-runner</artifactId>
       <version>1.3.5</version>
-      <scope>tests</scope>
+      <scope>test</scope>
       <exclusions>
         <exclusion>
           <groupId>commons-beanutils</groupId>

